### PR TITLE
Describe vision of probatus

### PR DESCRIPTION
**Pull request by Matgrb**

### CODE REVIEW

File: VISION.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## The Principles
+
+The main principles that drive development of `probatus` are the following
+
+- Usefulness - any tool that we build should be useful for a broad range of users,
+- Simplicity - simple to understand and analyse steps over state-of-the-art,
+- Usability - the developed functionality must be have good documentation, consistent API and work flawlessly with scikit-learn compatible models,
+- Reliability - the code that is available for the users should be well tested and reliable, and bugs should be fixed as soon as they are detected.
+
+## The Roadmap
```
Comment (timvink) [inline VISION.md:26]: I would use milestones or a pinned ROADMAP issue for this, which keeps the vision to be more static.

File: VISION.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+`probatus` has started as a side project of Data Scientists at ING Bank. 
+It consists of tools that help in analysing binary classification models and the data used to develop them.
+We have decided to open-source it, in order to share the tools and enable collaboration with the Data Science community.
+
+Our main goals are:
+
+- Continue maintaining the tools that we have built, and make sure that they are well documented and tested
+- Continuously extend functionality available in the package
+- Build a community of users, which use the package in day-to-day work and learn from each other, while contributing to probatus
```
Comment (timvink) [inline VISION.md:15]: Sound good, but could be more specific. Can we provide any more guidance on the scoping of the package other than 'tools to help analyze binary classifcation models and data used to train them' ?

**Review by timvink — CHANGES_REQUESTED** [review]
Some small remarks

**Comment by Matgrb** [comment]
Implemented the comments, what do you think?

**Comment by timvink** [comment]
Looks good to me, helps to give some direction & scope.

**Review by timvink — APPROVED** [review]

**Pull request merged**